core/src/main/scala/kafka/server/ReplicaManager.scala

```diff
@@ -1755,6 +1765,78 @@ class ReplicaManager(val config: KafkaConfig,
     partitionsToMakeFollower
   }
 
+  private def updateTopicIdForFollowers(controllerId: Int,
+                                        controllerEpoch: Int,
+                                        partitionStates: Map[Partition, LeaderAndIsrPartitionState],
+                                        correlationId: Int,
+                                        responseMap: mutable.Map[TopicPartition, Errors],
+                                        topicIds: String => Option[Uuid]) : Set[Partition] = {
```

Comment:
Is `topicIds` necessary? I suppose that we could get the `topicId` from the `Partition`, isn't it?

Reply:
I think you are right. makeFollowers needs it to assign the topic ID to the log for the first time, but for this PR, we've assigned it above.